src/core/properties/propertypresetmanager.cpp

```diff
@@ -55,6 +55,9 @@ bool PropertyPresetManager::loadPreset(const std::string& name, Property* proper
         // We deserialize into a clone here and link it to the original to only set value not
         // identifiers and such.
         auto temp = std::unique_ptr<Property>(p->clone());
+        // Since Observers are copied when cloning we need to blockNotifications to prevent change
```

Comment:
Not part of this PR?
We should remove the observers instead?

Reply:
I wanted to get this fix in, was too lazy to create a separate PR for it (it was already in my local branch). I do not want to touch the observer interface.

Reply:
But stopping blocking notifications won’t solve the core issue?

Reply:
Well, according to discussions in https://github.com/inviwo/inviwo/issues/594 it is not an issue and works as expected ;)